### PR TITLE
Container Apps Env Upgrade to latest version 2023-05-01

### PR DESCRIPTION
**Pull request by rajeshkaremane**
# Description

### Pipeline references

| Pipeline |
| - |
| |

## Type of Change

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Update to documentation

## Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings
- [x] My code follows the style guidelines of this project
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (readme)
- [x] I did format my code

### CODE REVIEW

**Comment by ChrisSidebotham** [comment]
Hi @rajeshkaremane - Thank you for your contribution. Can I ask if you have been able to test the workflow/module and are able to provide evidence of successful execution & Deployment?

File: modules/app/managed-environment/main.bicep [hunk 1]
```diff
@@ -17,13 +17,6 @@ param tags object?
 @description('Optional. Array of role assignments to create.')
 param roleAssignments roleAssignmentType
 
-@allowed([
-  'Consumption'
-  'Premium'
-])
-@description('Optional. Managed environment SKU.')
-param skuName string = 'Consumption'
```
Comment (AlexanderSehr) [inline modules/app/managed-environment/main.bicep:25]: How come the SKU was removed? Did this parameter become rundant?

Comment (rajeshkaremane) [reply modules/app/managed-environment/main.bicep:25]: Hi, yes the SKU name has been removed from the latest API
https://learn.microsoft.com/en-us/azure/templates/microsoft.app/change-log/managedenvironments

**Comment by rajeshkaremane** [comment]
> Hi @rajeshkaremane - Thank you for your contribution. Can I ask if you have been able to test the workflow/module and are able to provide evidence of successful execution & Deployment?

Hi,
I have executed our internal pipeline.

**Comment by rajeshkaremane** [comment]
> > > Hi @rajeshkaremane - Thank you for your contribution. Can I ask if you have been able to test the workflow/module and are able to provide evidence of successful execution & Deployment?
> > 
> > 
> > Hi,
> > I have executed our internal pipeline.
> 
> Hi @rajeshkaremane, can you evidence the output please and I'm happy to approve :)

Hi,

Please find attached the local test results.
Test 1
Test 1 - Resources
Test - 2
Test -2 Resources
Test -3
Test-3 Resources

Fixed minor issues in the dependency

**Pull request merged**